forge-game/src/main/java/forge/game/player/PlayerProperty.java

```diff
@@ -202,6 +202,20 @@ public static boolean playerHasProperty(Player player, String property, Player s
                     return false;
                 }
             }
+        } else if (property.startsWith("CardsDrawnThisTurn")) {
+            String comparator = property.length() > 18 ? property.substring(18, 20) : "GE";
+            int y = property.length() > 18 ? AbilityUtils.calculateAmount(source, property.substring(20), spellAbility)
+                    : 1;
+            if (!Expressions.compare(player.getNumDrawnThisTurn(), comparator, y)) {
+                return false;
+            }
+        } else if (property.startsWith("LandsEnteredThisTurn")) {
+            String comparator = property.length() > 20 ? property.substring(20, 22) : "GE";
+            int y = property.length() > 20 ? AbilityUtils.calculateAmount(source, property.substring(22), spellAbility)
+                    : 1;
+            if (!Expressions.compare(player.getLandsPlayedThisTurn(), comparator, y)) {
```

Comment:
Lands entered and Lands played is not the same

Reply:
Good call – this new version handles that nicely.

Stuff like Arboreal Grazer where the lands are played still count